torch/utils/data/datapipes/dataframe/dataframe_wrapper.py

```diff
@@ -0,0 +1,99 @@
+try:
+    import pandas  # type: ignore[import]
+
+    # pandas used only for prototyping, will be shortly replaced with TorchArrow
+    WITH_PANDAS = True
+except ImportError:
+    WITH_PANDAS = False
+
+
+class PandasWrapper:
+    @classmethod
+    def create_dataframe(cls, data, columns):
+        if not WITH_PANDAS:
+            Exception("DataFrames prototype requires pandas to function")
+        return pandas.DataFrame(data, columns=columns)
+
+    @classmethod
+    def is_dataframe(cls, data):
```

Comment:
this will crash if pandas not installed, should be 
```
if not WITH_PANDAS:
  return False
```

Reply:
Good catch thank you!